readme.md

```diff
@@ -196,20 +226,3 @@ After successfully building it, you will find firmware images at
 - .pio/build/heltec/firmware.elf
 
 
-## Binär-Datei `hani-mandl.bin`
-
-Die Datei `hani-mandl.bin` wurde mit folgenden Parametern für das Board Heltec ESP32 Arduino > Wifi Kit 32 compiliert: 
-
-```
-#define HARDWARE_LEVEL 2        // 1 = originales Layout mit Schalter auf Pin 19/22/21
-                                // 2 = Layout für V2 mit Schalter auf Pin 23/19/22
-#define SERVO_ERWEITERT         // definieren, falls die Hardware mit dem alten Programmcode mit Poti aufgebaut wurde oder der Servo zu wenig fährt
-                                // Sonst bleibt der Servo in Stop-Position einige Grad offen! Nach dem Update erst prüfen!
-#define ROTARY_SCALE 2          // in welchen Schritten springt unser Rotary Encoder. 
-                                // Beispiele: KY-040 = 2, HW-040 = 1, für Poti-Betrieb auf 1 setzen
-#define USE_ROTARY              // Rotary benutzen
-#define USE_ROTARY_SW           // Taster des Rotary benutzen
-```
-
-Eine Anleitung zum Flashen der Binär-Datei gibt es unter
-http://hanimandl.de/2020/12/23/firmware-binary-flashen/
```

Comment:
Diesen Bereich, der in der README hinzugekommen ist, bitte nicht entfernen. Vielen Dank!